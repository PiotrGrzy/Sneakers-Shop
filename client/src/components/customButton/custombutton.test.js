import React from 'react';
import CustomButton from './CustomButton';
import renderer from 'react-test-renderer';
test('renders snapshot', () => {
  const component = renderer.create(
    <CustomButton className="test">Test Button</CustomButton>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
