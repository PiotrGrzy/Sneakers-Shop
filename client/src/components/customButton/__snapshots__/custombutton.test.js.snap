// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot 1`] = `
<button
  className="test"
>
  Test Button
</button>
`;
